executor/aggregate_test.go

```diff
@@ -237,11 +237,6 @@ func (s *testSuite) TestAggregation(c *C) {
 	result = tk.MustQuery("SELECT COALESCE ( + 1, cor0.col0 ) + - CAST( NULL AS DECIMAL ) FROM t2, t1 AS cor0, t2 AS cor1 GROUP BY cor0.col1")
 	result.Check(testkit.Rows("<nil>", "<nil>"))
 
-	result = tk.MustQuery("select count(*) from information_schema.columns")
```

Comment:
why it's removed?

Reply:
It's removed in 98b96cc769160fb592b4cfc1f8a554c8751a6224